src/utils/channel_utils.js

```diff
@@ -569,13 +569,13 @@ export function sortChannelsByDisplayNameAndMuted(locale: string, members: Relat
 
 export function sortChannelsByRecency(lastPosts: RelationOneToOne<Channel, Post>, a: Channel, b: Channel): number {
     let aLastPostAt = a.last_post_at;
```

Comment:
Note that `channellast_post_at` does not change whenever there's changes on reaction to a post. It updates on new post or post edit (but not when updating has_reactions of a post). Therefore, it's safe to compare it against `post.create_at` when comparing for channel's recency.